### PR TITLE
Bringing namespace samples up to standards

### LINKED ISSUE #113: Bring appengine/multitenancy up to standards

### PR DESCRIPTION
**Pull request by jonparrott**
Fixes #113 and then some.

/cc @waprin.

### CODE REVIEW

File: appengine/multitenancy/README.md [hunk 1]
```diff
@@ -1,56 +1,13 @@
...
-repos](https://github.com/GoogleCloudPlatform) for sample applications and
-scaffolding for other python frameworks and use cases.
+### Running the sample
 
-## Run Locally
-1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/), including the [gcloud tool](https://cloud.google.com/sdk/gcloud/), and [gcloud app component](https://cloud.google.com/sdk/gcloud-app).
-2. Setup the gcloud tool.
+You can then run the sample on your development server:
+        
+      $ dev_appserver.py .
```
Comment (waprin) [inline appengine/multitenancy/README.md]: Why are you reverting back to this? While I think it's somewhat kludgey to have to download both SDKs I think it's still what we're recommending for now.

Comment (jonparrott) [inline appengine/multitenancy/README.md]: We have to be consistent with docs:
https://cloud.google.com/appengine/docs/python/tools/uploadinganapp

gcloud is still in preview.

**Comment by jonparrott** [comment]
@jerjou can you take a look? I need to prepare the change for the docs.

**Comment by waprin** [comment]
LGTM

**Pull request merged**